tests/resources/annotator/invalid_ids_on_ervery_element.cellml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this "typo in filename" file actually used anywhere?

Reply:
I don't believe it is.

Reply:
b0d2865